src/screens/LoginPage/LoginPage.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render, screen, within } from '@testing-library/react';
+import LoginPage from 'screens/LoginPage/LoginPage';
+
+describe('Testing the LoginPage', () => {
+  test('should show the text on the website', () => {
+    render(<LoginPage />);
+    expect(screen.getByText('Welcome Back')).toBeInTheDocument();
+    expect(screen.getByText('Login to your account')).toBeInTheDocument();
+  });
+});
```

Comment:
A test is still required to be written for the form even though the backend has not been written as yet. This article may provide some insights on how this could be achieved - [Testing Forms in React using Enzyme and Jest](https://dev.to/austinbh/testing-forms-in-react-using-enzyme-and-jest-2983l)

Reply:
done sir